server/data/index.ts

```diff
@@ -3,10 +3,10 @@
  * Do appinsights first as it does some magic instrumentation work, i.e. it affects other 'require's
  * In particular, applicationinsights automatically collects bunyan logs
  */
-import { buildAppInsightsClient, initialiseAppInsights } from '../utils/azureAppInsights'
+import AppInsightsUtils from '../utils/appInsightsUtils'
```

Comment:
Is it worth seeing if you can import/export from the utils index now that it's a class? It didn't work before, but this could have fixed it?

Reply:
As discussed, will leave this for now.